src/internal.d.ts

```diff
@@ -10,7 +10,7 @@ export interface FunctionalComponent<P = {}> extends preact.FunctionComponent<P>
 export type ComponentFactory<P> = preact.ComponentClass<P> | FunctionalComponent<P>;
 
 export interface PreactElement extends HTMLElement {
-	_prevVNode?: VNode<any> | null;
+	_children?: VNode<any> | null;
```

Comment:
It looks to me from a skim of the rest of the code that `_children` can be a VNode or an array or `null`, is that the case?

Reply:
My intention was to make it always an array (or null before it is defined) so if that's not true it is likely a bug or an oversight. I'll double check